connector_importer/components/odoorecord.py

```diff
@@ -124,16 +126,23 @@ def odoo_write(self, values, orig_values):
         odoo_record = self.odoo_find(values, orig_values).with_context(
             **self.write_context()
         )
-        self.odoo_pre_write(odoo_record, values, orig_values)
-        # TODO: remove keys that are not model's fields
-        odoo_record.write(values.copy())
+        # copy values to not affect original values (mainly for introspection)
+        values_for_write = values.copy()
```

Comment:
Not better to use `copy.deepcopy` here as `values` could contain some lists (e.g. of tuples `[(4, 0, ...)]`)?

Reply:
Checked with @simahawk : not so important